src/main/scala/scalapb/json4s/JsonFormat.scala

```diff
@@ -417,6 +418,10 @@ object JsonFormat {
       case JString(str) => Timestamps.parseTimestamp(str)
       case _ => throw new JsonFormatException("Expected a string.")
     })
+    .registerWriter((m: FieldMask) => JString(m.paths.mkString(",")), jv => jv match {
```

Comment:
can you add parsing too?

Reply:
Hi, `registerWriter` takes `writer` and `parser` as parameters. In test I also check parsing. I thought it's covered. Could you please give me more hints what I'm missing.

Reply:
@thesamet could you please explain what you mean by "can you add parsing too?" and what is missing in this pr to be merged.

Reply:
Can you use registerMessageFormatter so we have both json->scala and scala->json. Note that `[this PR](https://github.com/scalapb/ScalaPB/pull/470/files)` to ScalaPB's runtime has added JSON conversion methods, so they should be used here.